thinc/backends/numpy_ops.pyx

```diff
@@ -61,20 +61,25 @@ class NumpyOps(Ops):
             raise ValueError("BLIS support requires blis: pip install blis")
 
     def asarray(self, data, dtype=None):
+        array = None
```

Comment:
Unless there's a related check of some sort for a failure, this seems unnecessary.

```suggestion
```